public_html/wp-content/plugins/wc-fonts/wc-fonts.php

```diff
@@ -48,18 +49,22 @@ public function wp_head_google_web_fonts() {
 			return;
 		}
 
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		printf( '<style>%s</style>', $this->options['google-web-fonts'] );
 	}
 
 	/**
 	 * Provides the <head> output for Font Awesome
 	 */
 	public function wp_head_font_awesome() {
-		if ( empty( $this->options['font-awesome-url'] ) ) {
-			return;
+		if ( $this->options['font-awesome-url'] ) {
```

Comment:
I'm getting a `Notice: Undefined index: font-awesome-url` (and same for `font-awesome-kit`). This should probably either be wrapped in a `! empty()` call, or prefill all the correct keys up in the `init()` method.

Reply:
Yep, I forgot `empty` has that protection, thanks for the catch